src/ai/backend/manager/scheduler/fifo.py

```diff
@@ -34,27 +34,34 @@ def key_by_requested_slots(
         if k in unused_slot_keys and v > Decimal(0):
             num_extras += 1
 
-    sorted_agent_selection_order = sorted(
-        requested_slots.data.keys(), key=lambda item: get_slot_index(item, agent_selection_order)
+    for requested_slot_key in sorted(requested_slots.data.keys(), reverse=True):
+        first = requested_slot_key.split(".")[0]
```

Comment:
Conventionally we have called this string as `device_name`.
Let's follow the same convention.
(ref: https://github.com/lablup/backend.ai/blob/08c81ca48/src/ai/backend/agent/alloc_map.py#L127)